app/audit/models/WhatYouOweResponseAuditModel.scala

```diff
@@ -29,7 +29,7 @@ case class WhatYouOweResponseAuditModel(user: MtdItUser[_],
                                         whatYouOweChargesList: WhatYouOweChargesList,
                                         R7bTxmEvents: Boolean, dateService: DateServiceInterface) extends ExtendedAuditModel {
 
-  val currentTaxYear = user.incomeSources.getCurrentTaxEndYear(dateService)
+  val currentTaxYear = user.incomeSources.currentTaxYearEnd(dateService)
```

Comment:
* 'currentTaxYear is redundant property which we can drop and call corresponding function from dateService where needed;

Reply:
Done